web/theme.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:html';
+
+void init() {
+  var theme = document.getElementById('theme') as InputElement;
+
+  var elementB = document.body;
```

Comment:
We can just call this variable something like `body` or `bodyElement`. We try not to abbreviate too much.